tests/test_fields.py

```diff
@@ -412,6 +412,23 @@ def __init__(self, list):
         field = fields.List(fields.String, attribute='foo')
         self.assertEquals(['a', 'b', 'c'], field.output('list', obj))
 
+    def test_list_with_scoped_attribute_on_dict_or_obj(self):
+        class TestObject(object):
+            def __init__(self, list):
+                self.bar = list
```

Comment:
Absolutely - I was just following the pattern I thought I saw in the tests...perhaps I misread them a bit.